modules/runner/src/main/java/org/apache/ignite/internal/app/IgniteImpl.java

```diff
@@ -31,20 +32,28 @@
     /** Vault manager */
     private final VaultManager vaultManager;
 
+    private final SqlQueryProcessor qryProc;
+
     /**
      * @param tableManager Table manager.
      * @param vaultManager Vault manager.
+     * @param qryProc Query processor.
      */
-    IgniteImpl(IgniteTables tableManager, VaultManager vaultManager) {
+    IgniteImpl(IgniteTables tableManager, VaultManager vaultManager, SqlQueryProcessor qryProc) {
         this.distributedTableManager = tableManager;
         this.vaultManager = vaultManager;
+        this.qryProc = qryProc;
```

Comment:
Let's rename qryProc -> qryEngine to be consistent with getter method name.

Reply:
fixed